### PR TITLE
Enable theme-specific translations

### PR DESCRIPTION
**Pull request by Badlapje**
This PR will allow for theme-specific translations.  There are two fallbacks:

1. first the translations in te base theme
2. the translations in languages.

Skeune specific translations have been extracted.  Openconext specific translations can be extracted once the redesign is finished.

### CODE REVIEW

File: theme/base/scaffold/translations/messages.pt.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+
+/** Put all theme-specific Polish translations here.  */
```
Comment (MKodde) [inline theme/base/scaffold/translations/messages.pt.php:3]: ~Polish~ Portuguese :sunglasses:

File: theme/base/translations/messages.nl.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?php
+return [
+    // General
+    'logo'  => 'logo',
```
Comment (MKodde) [inline theme/base/translations/messages.nl.php:4]: Did you paste the translations here intentionally? I recon you want them in the `skeune` Dutch translations?

File: app/config/config.yml [hunk 3]
```diff
@@ -33,6 +33,8 @@ framework:
         fallbacks: ["%locale%"]
         paths:
           - '%kernel.root_dir%/../languages'
+          - '%kernel.root_dir%/../theme/base/translations'
+          - '%kernel.root_dir%/../theme/%theme.name%/translations'
```
Comment (MKodde) [inline app/config/config.yml:37]: This should work just fine! Maybe to be bulletproof, add an instruction in the theme README on the fact that the messages must be present in order for the theme to work.

File: theme/skeune/translations/messages.pt.php [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?php
+return [
+    'consent_h1_text' => 'Wien ester ier den boas?',
+    'suite_name' => 'Skeune theme',
+];
```
Comment (MKodde) [inline theme/skeune/translations/messages.pt.php:5]: I suggest pasting the English translations here.

Also:

> Den boas zit in de stafvergoaderinge altijd moar te kloage dat hij te weinig gerespecteerd wordt. Tsanderdoags brengt hij een plakoatse mee woar dat er opstoat :

> ‘Ik ben hier den boas!’ en hij plakt dat op de deure van zijnen bureau.

> Oast hij tsnoens werekomt van t’ ete hangt er een papierke aan zijn deure. Der stoat op:

> ‘Eu vrèwe hee gebeld. Ze wilt heur plakoat were!’

**Review by MKodde — COMMENTED** [review]
Nice work, creating theme specific translations. It woudl be even better if they where optional, but that would be a somewhat bigger challenge.

I found some issues, please review them.

File: theme/base/scaffold/translations/messages.pt.php (hunk 1 above)
Comment (Badlapje) [reply theme/base/scaffold/translations/messages.pt.php:3]: dumtiedum 😇

File: theme/base/translations/messages.nl.php (hunk 2 above)
Comment (Badlapje) [reply theme/base/translations/messages.nl.php:4]: wel spotted ... pasted them in the wrong folder indeed, rectified!

File: theme/skeune/translations/messages.pt.php (hunk 4 above)
Comment (Badlapje) [reply theme/skeune/translations/messages.pt.php:5]: done.  And 🤣

File: app/config/config.yml (hunk 3 above)
Comment (Badlapje) [reply app/config/config.yml:37]: Good one 💯 , added clarification in the Readme

File: theme/README.md [hunk 5]
```diff
@@ -83,11 +84,12 @@ The script must be run on the php-fpm instance on your CI environment as it also
 
 If you want to make your own theme it should be as a subfolder of the theme folder.  Name it however you want.
 
-Your theme **must** have the following structure:
+Your theme **must** have the following structure (or the build will fail):
 - images
 - javascripts: this needs to contain an `application.js` file, which is the entrypoint for all js loaded by your theme.
 - stylesheets: this needs to contain an `application.scss` file, which is the entrypoint for all css loaded by your theme.
 - templates: you only need to put templates here you want to override.
+- translations: you only need to put files there if there are actual theme-specific translations.
```
Comment (MKodde) [inline theme/README.md:92]: :partying_face:

File: theme/scripts/build.js [hunk 6]
```diff
@@ -44,5 +43,6 @@ function executeShellCommand(command) {
         }
 
         console.log(stdOut);
+        return process.exit(0);
```
Comment (MKodde) [inline theme/scripts/build.js:46]: Thanks, that bothered me too! Thanks for fixing this!

**Review by MKodde — APPROVED** [review]
Excellent, lets merge!

**Pull request merged**